src/components/Input/InputStyled.js

```diff
@@ -46,7 +38,7 @@ export const LabelStyled = styled.label.attrs(() => ({
     width: 100%;
     height: 15px;
 
-    background: ${BackgroundColor};
+    background: ${props => props.theme.BackgroundColor};
```

Comment:
we could simplify all linke like this to: 
${({theme}) => theme.BackgroundColor};